infra/lib/charts/prow-secrets.ts

```diff
@@ -4,46 +4,88 @@ import * as kplus from 'cdk8s-plus-20';
 import { PROW_NAMESPACE, PROW_JOB_NAMESPACE } from '../test-ci-stack';
 
 export interface ProwSecretsChartProps {
-  readonly botPersonalAccessToken: string;
-  readonly webhookHMACToken: string;
+  readonly githubPersonalAccessToken: string;
+  readonly githubAppId: string;
+  readonly githubAppClientId: string;
+  readonly githubAppPrivateKey: string;
+  readonly githubAppWebhookSecret: string;
 }
```

Comment:
Might be worth it to rename this whole class to `ProwGithubSecretsChart` and remove the word `github` from appearing 100x

Reply:
Done